artichoke-backend/src/lib.rs

```diff
@@ -126,6 +126,7 @@ mod eval;
 pub mod exception_handler;
 pub mod extn;
 pub mod ffi;
+pub mod hash;
```

Comment:
there's nothing here to export, there's only a trait implementation, so let's not make this `pub` like `globals` and `intern` below.

```suggestion
mod hash;
```